tests/pytorch/test_optim.py

```diff
@@ -0,0 +1,46 @@
+import torch as th
+import backend as F
+
+from dgl.backend.pytorch.sparse_emb import NodeEmbedding
+from dgl.backend.pytorch.sparse_optim import SparseAdam, SparseAdagrad
```

Comment:
Should avoid this namespace even in unittest. Try use user-facing ns as much as possible.